android/src/main/kotlin/io/ente/photos/media_extension/MediaExtensionPlugin.kt

```diff
@@ -145,6 +145,9 @@ class MediaExtensionPlugin : FlutterPlugin, MethodCallHandler, ActivityAware,
                     resAction = IntentAction.valueOf("EDIT")
                 }
                 Intent.ACTION_VIEW -> {
+                    if(data != null) {
```

Comment:
Missing space between `if` and the `(`. You can setup your IDE to auto-format on save :)

Reply:
Done